client/tools/mgr-cfg/mgr-cfg.changes

```diff
@@ -1,3 +1,4 @@
+- Remove unused rhnsd/mgr-daemon
```

Comment:
You are missing an extra line, as this is the first changelog after last header.
```suggestion
- Remove unused rhnsd/mgr-daemon

```